setup.cfg

```diff
@@ -58,12 +58,9 @@ install_script = scripts/rpm-install.sh
 [flake8]
 exclude = build,.git,.tox,./tests/.env
 ignore = W504,W601
-max-line-length = 119
+max-line-length = 88
```

Comment:
We should chop `ignore` and add:
```
extend-ignore = E203
```
see [Black's docs](https://black.readthedocs.io/en/stable/guides/using_black_with_other_tools.html#flake8).